committing just changed files (#31)
ottrproject/cheatsheets: google_analytics.Rmd

---
output:
  html_document:
    number_sections: false
    includes:
      in_header: ottr_favicon.html
css: css/OTTR_style.css
---

```{r, echo=FALSE, results='asis'}
ottrpal::borrow_chapter(
  doc_path = "chunks/create_ottr_header.md",
  tag_replacement = list(
    "{TITLE}" = "OTTR Google Analytics", "{SUBTITLE}" =  "Integration with your repository",

>         "{PATH_TO_PNG}" = "https://github.com/ottrproject/cheatsheets/blob/main/pngs/ottr_quarto_website.png?raw=true"

Reviewer: ```suggestion
        "{PATH_TO_PNG}" = "https://github.com/ottrproject/cheatsheets/blob/main/pngs/google_analytics.png?raw=true"
```

Reviewer: this is where the png will be

<input type="checkbox"> Click **Test** on the stream page, then open your live course URL and check that you appear as a visitor in Google Analytics

<input type="checkbox"> Click the **Home button** — it should say **"Data collection is active"**

<input type="checkbox"> Check the **Reports** tab (bar chart icon, upper left) — opening your course in a browser should show you as 1 user in the last 30 minutes

<hr>

# Troubleshooting

| Issue | What to Check |
|----|----|
| No data showing in GA | Make sure the **code chunk** in `GA_Script.html` was copied correctly and contains the right ID (`G-XXXXXXXXXX`) |
| Data stream shows no traffic | Check that the **URL** in the stream matches your course URL — watch for `https` vs `http` |
| Old template not rendering GA | Manually run the `render_all` workflow; confirm both `_output.yml` and `GA_Script.html` are updated and filenames are consistent |
| Render not triggering | Ensure `GH_PAT` is set up for the repo (with `actions` and `repo` permissions), not just the organization |

```{r, echo=FALSE, results='asis'}
ottrpal::borrow_chapter(
  doc_path = "chunks/create_ottr_footer.md",
  tag_replacement = list(

>     "{AUTHORS}" = "Kate Isaac wrote the content and created the original styling, Carrie Wright did the css styling and Carrie Wright and Candace Savonen did the publishing engineering for the cheatsheet. Padmashri Saravanan from [BioDataSage](https://www.biodatasage.com/) worked with Carrie Wright to review and improve the cheatsheet."

Reviewer: ```suggestion
    "{AUTHORS}" = "Padmashri Saravanan from [BioDataSage](https://www.biodatasage.com/) created the cheatsheet content. Carrie Wright reviewed the material. Kate Isaac created the original styling, Carrie Wright did the css styling and Carrie Wright and Candace Savonen did the publishing engineering for the cheatsheet. "
```

Reviewer: updating who did what for this one
